### DISCUSSION TITLE
Patch2Self killed with different patch radius

### DISCUSSION THREAD

**Discussion by staunton-golding** [dipy/dipy#3010] [Preprocessing]
Hello,
I am wanting to adjust the size of the patch radius, but when I attempt to do so, Patch2Self is able to denoise the b0 volumes but then gets killed. The script works when I use default patch_radius. This is my line:

denoised_arr = patch2self(data,bvals,verbose = True, patch_radius = [1, 1, 1])

Any ideas why its getting killed? Thanks!

**Comment by ShreyasFadnavis** [comment]
Hi @staunton-golding ! Thanks for reporting this. It seems like you are running into memory issues. 

Can I ask what is the dimensionality of your data? Using a higher patch radius may not be super useful if the data is big enough :)

**Reply by staunton-golding** [reply]
Hi Shreyas,

Thanks for the quick reply! The data is 140x140x140x51. 

Another question / if you have any thoughts - I am using it on data that has already been eddy and motion corrected. When looking at the patch radius=0 patch2self residuals, the background voxels are completely uniform. I imagine that must be a result of this particular eddy correction / motion correction method and not a result of patch2self right?
